Makefile

```diff
@@ -42,23 +47,19 @@ _IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
 _APB_IMAGE = $(_DOCKER_REGISTRY):5000/$(_TEST_PROJECT)/$(DEV_APB_IMAGE_NAME)
 _OPENSHIFT_USERNAME = developer
 _OPENSHIFT_PASSWORD = developer
-_TESTRUNNER_PORT = 80
 else
 _DOCKER_REGISTRY = $(OPENSHIFT_ONLINE_REGISTRY)
 _IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
-_TESTRUNNER_PORT = 80
 endif
 
-_DEV_IMAGE_STREAM = $(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
-
 # This username and password is hardcoded (and base64 encoded) in the Ansible
 # Service Broker template
 _ANSIBLE_SERVICE_BROKER_USERNAME = admin
 _ANSIBLE_SERVICE_BROKER_PASSWORD = admin
 
 start-openshift-with-catalog:
 	@echo "---- Starting OpenShift ----"
-	oc cluster up --service-catalog
+	oc cluster up --version="$(OPENSHIFT_VERSION)"
```

Comment:
When you explicitly mention the version, does it start the cluster with the service-catalog?

Reply:
Ouch, it seems we can not use this trick because of the API server incompatibility: https://github.com/openshift/origin/issues/18288#issuecomment-363516099.

I need to find another way...

Reply:
So we basically have 3 options:

Option #1 - use a downgraded `oc` binary for the tests. That should work fine, we just need to add a new target to the `Makefile` called `download-oc-binary`.

Option #2 - Use `oc cluster up --version="$(OPENSHIFT_VERSION)"` without Service Catalog for testing. This may lead to some weird errors due to API incompatibility.

Option #3 - Simply remove all use of image streams and always refer to an absolute image name in the templates.

Since the AOS Team is still discussing this, I would follow the simplest route. I would add additional target called `download-oc-binary`. That target would download OCP 3.7.2 and put in the working directory if requested. Otherwise it would create a symbolic link to the `oc` client already installed on the OS. This way we could rewrite all calls to the `oc` binary from the `Makefile` from `oc` to `./oc`.

Reply:
I don't have strong views either way, but option number 1 sounds good to me.

Reply:
OK. Option 1) sounds good.